jsf-showcase-webapp/src/main/webapp/WEB-INF/component/f/convertdatetime/general/convertDateTime.xhtml

```diff
@@ -24,4 +24,4 @@
 		</h:form>
 	</ui:define>
 
-</ui:composition>
\ No newline at end of file
+</ui:composition>
```

Comment:
Maybe I'm wrong but I don't see any changes on line 27. However the format pattern of the date seems rather aesthetic to me.